frontend/package.json

```diff
@@ -25,6 +24,7 @@
     "@eslint/js": "^9.22.0",
     "@types/react": "^19.0.10",
     "@types/react-dom": "^19.0.4",
+    "@types/spotify-web-api-node": "^5.0.11",
```

Comment:
I decided to go with the same package as the backend to get the types. To hopefully keep prevent any inconsistencies.